From 8adb924b653124445afff71dc61116907c01073d Mon Sep 17 00:00:00 2001
From: Steve Kondik <shade@chemlab.org>
Date: Sun, 20 May 2012 22:10:30 -0700
Subject: [PATCH 12/44] build: Stop installing the sample APN list

 * Sample APN list is always being installed, we don't want this in CM.

Change-Id: I74b97f3af545c4a7568d3001e1435c63cfbc7de8
Signed-off-by: Kshitij Gupta <kshitijgm@gmail.com>
---
 target/product/full_base_telephony.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/target/product/full_base_telephony.mk b/target/product/full_base_telephony.mk
index af4097d..8f3007a 100644
--- a/target/product/full_base_telephony.mk
+++ b/target/product/full_base_telephony.mk
@@ -24,7 +24,6 @@ PRODUCT_PROPERTY_OVERRIDES := \
     ro.com.android.dataroaming=true
 
 PRODUCT_COPY_FILES := \
-    device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml \
     frameworks/native/data/etc/handheld_core_hardware.xml:$(TARGET_COPY_OUT_VENDOR)/etc/permissions/handheld_core_hardware.xml
 
 $(call inherit-product, $(SRC_TARGET_DIR)/product/aosp_base.mk)
-- 
2.7.4

